28-04/task_2/task_2.rb

```diff
@@ -0,0 +1,10 @@
+require './cleanfile'
```

Comment:
we can use `require_relative` here, so we don't need to put `./` in a start of declaration